core/src/completed_data_sets_service.rs

```diff
@@ -1,3 +1,9 @@
+//! [`CompletedDataSetsService`] is a hub, that runs different operations on when a "completed data
+//! set", also known as a [`Vec<Entry>`] is received by the validator.
+//!
+//! Currently, it is driven by the [`crate::window_service`], sending [`CompletedDataSetInfo`]s via
```

Comment:
`[`crate::window_service`] , sending`-> `WindowService sends`

Reply:
The upside of saying `` [`crate::window_service`] ``, is that it is a link checked by `cargo doc`.  Changing it to `WindowService` removes the checking.
`WindowService` itself is not public, so it can not be included in the generated documentation.